src/pages/contributors/james-nord.adoc

```diff
@@ -0,0 +1,150 @@
+So I was working on a lot of plugins for CloudBees and sometimes having to fix bugs in open source.
+I've also attempted to maintain the acceptance test harness.
+
+Most recently, I've been driving the FIPS effort for our proprietary products, which means we have had to have FIPS hooks inside Jenkins.
+We've been working with the team and managing that to get enough into open source so that we can build on it.
+There's a JEP and various plugins that we have contributed towards the FIPS effort.
+CloudBees CI has now gone GA with support for FIPS 140, so the base work for that is obviously in the open source.
+However, it's not as easy to run and you need to do your own checks.
+
+== Why choose Jenkins over other projects?
+
+While I was working for NDS, the satellite cable company, as we were scaling up and out, we wanted to be able to allow users to create their own projects.
+However, we wanted them to essentially be rubber stamp templates.
+So it's like this: you're always going to use a Maven project, so the only thing that is different is where your source control is located and what email should be notified when something goes wrong.
+That was around the time that Kohsuke had just started his company InfraDNA.
+
+I believe the first time that I worked with Kohsuke, was the Nectar VMware autoscaling plugin which allowed us to have a pool of resources in VMware that were just virtual machines and just resetting to snapshots and assign them into builds.
+So we could say, for this build I need a virtual machine; here you go, here's an IP address, and we were good to go.
+I was amazed at the quickness in which he knocked out that plugin.
+While in Jerusalem, we mentioned something during a meeting and he came in the next day and said here you go, try this.
+It was almost out of nowhere and the collaboration with Kohsuke was amazing.
+
+Additionally, at the time it had a cutting edge UI and the test result reports were great.
+If you compared it to other stuff, it was a little table of pass/fail or a mess.
+Hudson allowed you to have a summary, drill into things, and show you the full stack trace and full logs right at the bottom of the builds, as opposed to having all the information in a massive sprawling page of mess.
+
+== What problems have you solved in Jenkins?
+
+The most recent thing is being able to support FIPS 140 for any federal people that want to use Jenkins and do it themselves.
+Up until a year ago, you couldn't really start Jenkins in a FIPS compliant way because the initial use password that it generates would be generated in something that was not an approved encryption standard.
+
+Another one of the things I did recently, which if you end up having to support Jenkins, is all of the plugin class loaders are now named.
+If something does go wrong and you get a stack trace, rather than having just the class, you have plugin name and class so you can immediately see which plugin is having an issue.
```

Comment:
```suggestion
If something does go wrong and you get a stack trace, rather than having just the class, you have the plugin name and class so you can immediately see which plugin is having an issue.
```